lms/templates/courseware/static_tab.html

```diff
@@ -6,6 +6,11 @@
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
 </%block>
+<%block name="js_extra">
+
+<%include file="../discussion/_js_body_dependencies.html" />
```

Comment:
@adampalay I had the same question in my mind but then i saw couple of html files in `courseware` are including that.

* [courseware/courseware-chromeless.html](https://github.com/edx/edx-platform/blob/4334eb9bd19e51e0712db9d3b021f3d66c5c458c/lms/templates/courseware/courseware-chromeless.html#L57)
* [courseware/courseware.html](https://github.com/edx/edx-platform/blob/5e972b2a66e214c53372bdb4d41a71215af9562f/lms/templates/courseware/courseware.html#L86)